R/generate_simulated_data.R

```diff
@@ -154,19 +154,19 @@ generate_simulated_data <- function(r_in_weeks = # nolint
                                       dist_matrix = as.matrix(
                                         dist(
                                           data.frame(
-                                            x = c(85, 37, 48, 7),
-                                            y = c(12, 75, 81, 96),
-                                            diag = TRUE,
-                                            upper = TRUE
-                                          )
+                                            x = c(85, 37, 36, 7),
+                                            y = c(12, 75, 75, 96)
+                                          ),
+                                          diag = TRUE,
+                                          upper = TRUE
                                         )
-                                      ),
-                                      phi = 25,
+                                      ) / 114.62984,
```

Comment:
This can be a separate issue, but probably worth writing a function that normalizes the distance matrix within the `generate_simulated_data()` function, and just allows the "user" of this function to pass in their distance matrix. Since this right now isn't completely clear what you are doing.

Reply:
See also https://github.com/CDCgov/ww-inference-model/pull/56#discussion_r1722360146